src/app/public/modules/datepicker/datepicker.component.ts

```diff
@@ -51,7 +51,7 @@ export class SkyDatepickerComponent implements OnDestroy {
   public dateChange = new EventEmitter<Date>();
   public maxDate: Date;
   public minDate: Date;
-  public startingDay: number;
+  public startingDay: number = 0;
```

Comment:
What do you think about setting the default in the calendar component, so that if `undefined` is passed (or set) from on-high, it will always resolve to `zero` when it reaches the calendar.

https://github.com/blackbaud/skyux-datetime/blob/master/src/app/public/modules/datepicker/datepicker-calendar.component.ts#L34

Reply:
The calendar component has its own spec, too.

Reply:
Good call. Done. 

I also noticed that with what I had that if the user actually bound to undefined we would still use that. My new version uses a getter/setter in order to ensure we use "0" if undefined is passed in.

You also may notice that I no longer have a test around this. This is because we really already had a test for this (line 131 of the datepicker-calendar spec); however, it was passing before because our test component was setting 0 by default. I have changed the test component to not send in a value unless specifically set in a test (which we do have 2 tests doing this).